reference/api/search.md

```diff
@@ -19,6 +19,10 @@ Other than the differences mentioned above, the two routes are strictly equivale
 
 Search for documents matching a specific query in the given index. The index [`uid`](/learn/core_concepts/indexes.md#index-uid) is required.
 
+:::note
+This route has a [non-customizable limit of 1000 results](/learn/advanced/known_limitations.md#maximum-number-of-results-per-search). If you want to scrap your database, you can use the [get documents endpoint](/reference/api/documents.md#get-documents) instead.
```

Comment:
`If you want to scrap your database, you can use the [get documents endpoint](/reference/api/documents.md#get-documents) instead.`
Should we mention this here? Or is the link to known limitations enough?

Reply:
I like it! Seems like a reasonably helpful tip.